tools/release-plugin.sh

```diff
@@ -12,128 +13,454 @@ BASE=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
 . "$BASE/tools/includes/changelogger.sh"
 . "$BASE/tools/includes/send_tracks_event.sh"
 
+VERSION_REGEX='^[0-9]+(\.[0-9]+)+(-.*)?$'
+CUR_STEP=0
+
+RELEASE_STEPS=(
+	'do_trunk_and_prelease_branch_prep'
+	'do_changelogs'
+	'do_readme'
+	'do_commit_changelog_and_readme'
+	'do_push_and_build'
+	'do_packagist_check'
+	'do_create_release_branches'
+	'do_create_prerelease_PR'
+	'do_final_instructions'
+)
+
 # Instructions
 function usage {
 	cat <<-EOH
-		usage: $0 [options] <plugin> [<version>] [<plugin> [<version>] ...]
-
-		Conduct a full release of specified plugins through release branch creation. Just the plugin name is fine, such as 'jetpack' or 'backup.' The version is optional, and if not specified, will be set to the next stable version.
+		Usage:
+		    $0 [-s <step>] <plugin> [<version>]
```

Comment:
I'd bring back the part that indicates multiple plugins can be specified
```suggestion
		    $0 [-s <step>] <plugin> [<version>] [<plugin> [<version>] ...]
```